### ISSUE TITLE
Associate finding explanation to finding ID in docs

### ISSUE THREAD

**Issue by lorengordon** [plus3it/watchmaker#1088]
Add a table to the findings index to make it easier for users to lookup a finding by the finding ID (CVE and STIG to start). Something like:

```
| Finding Explanation | Finding IDs                         |
|---------------------|-------------------------------------|
| <link to heading>   | <list of known-related finding IDs> |
```

Doc link: https://watchmaker.cloudarmor.io/en/stable/findings/index.html#
Source: https://github.com/plus3it/watchmaker/blob/develop/docs/findings/index.rst

**Comment by ferricoxide** [comment]
After scan-output review, opened two issues:

*  AIDE configuration is not correct &mdash; plus3it/ash-linux-formula#284
* Incorrect default firewalld zone &mdash; plus3it/ash-linux-formula#285

**Comment by ferricoxide** [comment]
Closed by #1095
